### ISSUE TITLE
Error: "File mode specification error: (void-function kbd-mode--show-macros?)"

### ISSUE THREAD

**Issue by kfogel** [kmonad/kbd-mode#11]
After loading `kbd-mode.el` (commit a7678e43c9eb71c), when I find a `.kbd` file into a buffer I get this error:

```quote
File mode specification error: (void-function kbd-mode--show-macros?)
```

It looks like this is a result of commit a7678e43c9eb unintentionally (?) reverting part of earlier commit a4c5f1c60ff39.  The earlier commit had renamed `kbd-mode--show-macros?` to `kbd-mode--show-macros-p` *and* changed the caller in `(define-derived-mode kbd-mode ...)`.  But then commit a7678e43c9eb reverted the change to the caller (commit a7678e43c9eb contains a lot of changes -- it appears to be a wholesale import of a lot of unrelated changes at one, and I guess this one slipped through accidentally).

**Comment by slotThe** [comment]
Closed via #12

**Comment by kfogel** [comment]
Thank you, @slotThe.
